client/src/components/install.jsx

```diff
@@ -68,13 +77,13 @@ function install(props) {
 
             <ul>
               <li>
-                You can run the bootloader by itself to play on Eden but I
-                suggest to use Ashita.
+                You can run the bootloader by itself to play on Eden, but
+                suggested use is with Ashita (or Windower).
```

Comment:
Only thought here is either one of these two. If we are officially supporting Windower do 1, else 2:
1. "suggested use is with Ashita or Windower."
2. "suggested use is with Ashita."

Reply:
2, ty